PayPalNative/src/main/java/com/braintreepayments/api/PayPalNativeOnActivityResumedCallback.java

```diff
@@ -0,0 +1,6 @@
+package com.braintreepayments.api;
+
+public interface PayPalNativeOnActivityResumedCallback {
```

Comment:
Let's add a doc string for this (see the doc string pattern for any other OnActivityResumed callback)

Reply:
this class is no longer needed, i forgot to delete it